CHANGELOG.rst

```diff
@@ -7,31 +7,36 @@ Change log
 BREAKING CHANGES
 ~~~~~~~~~~~~~~~~
 
-* **migrate to Python3.7** due to Python2 is no longer supported
-    Python3.7 is now :doc:`required <prerequisites>` for Universum to work
-* **disable entry points** and switch to ``python3.7 -m universum`` calls
-    to avoid possible collisions, especially when using :doc:`analyzers <code_report>`
+* **migrate to Python 3.7** due to Python 2 is no longer supported
+   Python3.7 is now :doc:`required <prerequisites>` for Universum to work; please switch ``print ""`` into
+   ``print("")`` in config files if needed
+* **disable entry points** (calling ``universum`` from command line) and switch to ``python3.7 -m universum`` calls
+   to avoid possible situations where wrong version of Universum is executed, especially when using
+   :doc:`analyzers <code_report>`, that are launched from inside an active Universum
 * **rename `_universum` to `universum`**
-    :class:`~universum.configuration_support.Variations` imports in configs need to be fixed
+   :class:`~universum.configuration_support.Variations` imports in configs need to be fixed: use
+   ``from universum.configuration_support import Variations`` instead of
+   ``from _universum.configuration_support import Variations``
 
 New features
 ~~~~~~~~~~~~
 
 * **github-handler:** add :doc:`a new Universum mode <github_handler>`
-    to serve as a `GitHub App <https://docs.github.com/en/developers/apps>`__. GitHub Applications,
-    for example, are needed to run checks on newly commmited changes, or on changes to be merged yet
+   to serve as a `GitHub App <https://docs.github.com/en/developers/apps>`__. GitHub Applications
+   are needed to run checks and report their status to GitHub for changes newly commmited or those to be merged yet.
```

Comment:
"for changes newly commmited or those to be merged yet" -> "for changes on review and for committed changes"